### PR TITLE
mission complete

### PR DESCRIPTION
**Pull request by yurii-filiushkin**
- [DEMO LINK](https://yurii-filiushkin.github.io/layout_calendar/)
- [TEST REPORT LINK](https://yurii-filiushkin.github.io/layout_calendar/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,6 +7,38 @@
...
+      <div class="calendar__day calendar__day--22"></div>
+      <div class="calendar__day calendar__day--23"></div>
+      <div class="calendar__day calendar__day--24"></div>
+      <div class="calendar__day calendar__day--25"></div>
+      <div class="calendar__day calendar__day--26"></div>
+      <div class="calendar__day calendar__day--27"></div>
+      <div class="calendar__day calendar__day--28"></div>
+      <div class="calendar__day calendar__day--29"></div>
+      <div class="calendar__day calendar__day--30"></div>
+      <div class="calendar__day calendar__day--31"></div>
```
Comment (marina-tilniak) [inline src/index.html]: `calendar__day--29` навіщо ці класи?

**Review by marina-tilniak — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 1 above)
Comment (yurii-filiushkin) [reply src/index.html]: Я так числа проставив
@for $index from 1 through 31 {
      &--#{$index}::before

Comment (yurii-filiushkin) [reply src/index.html]: Виправив

File: src/styles/blocks/calendar.scss [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  }
+
+  $days:
+    "mon" 0,
+    "tue" #{1 * ($day-width + $gap-value)},
+    "wed" #{2 * ($day-width + $gap-value)},
+    "thu" #{3 * ($day-width + $gap-value)},
+    "fri" #{4 * ($day-width + $gap-value)},
+    "sat" #{5 * ($day-width + $gap-value)},
+    "sun" #{6 * ($day-width + $gap-value)};
```
Comment (ivanchuk-yurii) [inline src/styles/blocks/calendar.scss:44]: it's better to calculate margins inside `@each` with `list.index`

**Review by ivanchuk-yurii — APPROVED** [review]

**Pull request closed without merging**